### ISSUE TITLE
Missing code highlight when tracing map literal

### ISSUE THREAD

**Issue by ingesolvoll** [jpmonettas/flow-storm#10]
Given a map like this:
```
{:request {:remote [:my-remote
                    {:prop1 prop1
                     :prop2 prop2}]
           :cb     cb}}
```
The debugger steps through and displays the prop1 and prop2 values in pprint window on the right. But symbols are not highlighted in code window on the left, the entire map form is highlighted.

**Comment by jpmonettas** [comment]
Will look into it as soon as flow-storm-debugger 0.3.0 is released (hopefully this weekend).

Thanks for reporting!

Cheers!

**Comment by jpmonettas** [comment]
I wasn't able to reproduce this in the latest version. Can you retry with `flow-storm-debugger 0.3.1` and `flow-storm 0.3.0` ?

Please reopen it if you are still seeing the issue.

**Comment by ingesolvoll** [comment]
With the latest version, the entire map is highlighted through the 5 steps, with only the last step rendering the total result. The first 4 steps renders nothing in the right frames. This is CLJS. See attached trace file.

[literal-map.txt](https://github.com/jpmonettas/flow-storm/files/5470388/literal-map.txt)

**Comment by ingesolvoll** [comment]
This is the test case 
```
(def a 1)
(def b 2)

#trace
    {:request {:remote [:my-remote
                        {:prop1 (+ a 2)
                         :prop2 (/ b 4)}]
               :cb     [:hey]}}

```

**Comment by ingesolvoll** [comment]
@jpmonettas By the way, I don't think I have permissions here to reopen the issue. Or am I misunderstanding the UI?

**Comment by jpmonettas** [comment]
@ingesolvoll I tested it  on Clojure and was working fine for me, will re test it in ClojureScript when I get some time, maybe the bug is only showing there?

**Comment by ingesolvoll** [comment]
Tested in CLJ repl now, same result

**Comment by ingesolvoll** [comment]
Could it be clojure version? I'm on 1.9. Seems unlikely.

**Comment by jpmonettas** [comment]
Hey, this was a nasty one but is fixed in flow-storm-debugger 0.3.3.  (https://github.com/jpmonettas/flow-storm-debugger/commit/ac8caf7f167e0aa347d58957ff8b478afa3f83a8)

Flow-storm is using zprint to pprint the code. By default zprint sorts maps keys, so for maps literals that are sorted the bug doesn't show, but when they are unsorted, the pprint version is different from the original and the coordinates doesn't match anymore.

Looks like I typed your first example a little different (sorted) and it was working for me.

Anyway, fixed now.

Cheers!

**Comment by ingesolvoll** [comment]
Confirming that is works for me now.
